docs/content/manual/pre-release/node/backing-image-on-a-down-node.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Backing Image on a down node
+---
+
+1. Update the settings:
+   1. Disable Node Soft Anti-affinity.
+   2. Set Replica Replenishment Wait Interval to a relatively long value.
+2. Create a backing image with one of the following URLs:
+```
+   https://longhorn-backing-image.s3-us-west-1.amazonaws.com/parrot.qcow2
+   https://longhorn-backing-image.s3-us-west-1.amazonaws.com/parrot.raw
+```
+3. Create and 2 volumes with the backing image and attach them on different nodes.
+4. Verify the backing image content then write random data in the volumes.
+5. Power off a node containing one volume. Verify that
+   - the related disk download state in the backing image will become `failed` once the download pod is removed by Kubernetes.
+   - the volume on the running node still works fine but is state `Degraded`, and the content is correct in the volume.
+   - the volume on the down node become `Unknown`.
+6. Power on the node. Verify
+   - the failed replica of the `Degraded` volume can be reused.
+   - the volume on the down node will be recovered automatically.
+   - the backing image will be recovered automatically.
```

Comment:
We should also check the random data written at step 4 by reducing the replica count to one and deleting the replicas from the always active nodes.

Reply:
Good idea. I will add this step to the test